### DISCUSSION TITLE
Super+Shift+Return not working

### DISCUSSION THREAD

**Discussion by saltbr1nger** [i3/i3#6152] [Q&A] [answered]
I'm trying to bind `exec firefox` to Super+Shift+Return, but it's not working.  Instead, it launches my terminal, which is bound to Super+Return.  Any advice for me?

edit: Nm, reboot fixed it.  Closing.

**Comment by orestisfl** [comment]
Paste your config file

**Reply by saltbr1nger** [reply, marked as answer]
#start a terminal
bindsym Mod4+Return exec alacritty

#launch browser
bindsym Mod4+Shift+Return exec firefox

**Reply by ghost-2e63** [reply]
try something like this

```
bindsym Mod4+Shift+Return exec --no-startup-id /usr/bin/firefox
```

if it doesn't work probably check if `firefox` can be run in from the terminal.

EDIT: nvm, just read the edit, that reboot fixed your issue.
